tests/helpers.py

```diff
@@ -96,3 +101,19 @@ def wrapper(*args: P.args, **kwargs: P.kwargs) -> T:
         return wrapper
 
     return decorator
+
+
+def next_hour(hours: int = 0, *, minutes: int = 0, days: int = 0) -> datetime.datetime:
```

Comment:
Maybe argument names should be `plus_hours`, `plus_minutes`, and `plus_days` since we always get the next hour, and these just add more to that? I would also make all of them keyword-only for clarity.